lib/auth/tls_test.go

```diff
@@ -1518,6 +1531,28 @@ func (s *TLSSuite) TestWebSessionWithApprovedAccessRequest(c *check.C) {
 
 	_, hasRole = mappedRole["test-request-role"]
 	c.Assert(hasRole, check.Equals, true)
+
+	// Test swtich back to default role and expiry.
```

Comment:
Can you add this as an additional test case? Usually, the old tests are not changed if the base functionality was not modified to render it invalid. It would be additionally helpful to have as minimal test case as possible, sometimes the tests grow to become cumbersome to read and know what's being tested.